packages/validator/src/commands/convertCmd.ts

```diff
@@ -0,0 +1,50 @@
+import fs from 'fs';
```

Comment:
Also, if I'm converting something I would expect the output (not the field `output` but the output of the command) to be the converted JSON. Or, at least to have an option output the result into a file.

Reply:
This is a definitely useful feature, added in aea2e9b